commit cb5ff43fb308c1dcb8d1eaf2935754ee840cbcee (refs/changes/46/146/2)
Author: Johanan Lai <lai.johanan@gmail.com>
Date:   2016-07-26T14:53:00-07:00 (3 years, 2 months ago)
    
    ADMINUI-2321 Data center should be immutable when editing user provisioning limits

diff --git a/www/js/components/pages/user/limits/form.jsx b/www/js/components/pages/user/limits/form.jsx
index 47a01ee2..b918ea98 100644
--- a/www/js/components/pages/user/limits/form.jsx
+++ b/www/js/components/pages/user/limits/form.jsx
@@ -172,7 +172,7 @@ var LimitsForm = React.createClass({
                     <div className="form-group">
                         <label className="col-md-4 control-label">Datacenter</label>
                         <div className="col-md-7">
-                            <Chosen className="form-control" placeholder="Select a Datacenter" value={this.state.datacenter} onChange={this.handleChangeDatacenter}>
+                            <Chosen className="form-control" placeholder="Select a Datacenter" value={this.state.datacenter} onChange={this.handleChangeDatacenter} disabled={this.props.initialLimit ? 'disabled' : ''}>
                             <option value=""></option>
                             {
                                 this.state.datacenters.map(function (dc) {
